src/components/DockApp.vue

```diff
@@ -188,26 +248,58 @@ export default {
   }
 }
 
+@keyframes bounce-horizontal-reverse {
+  0%,
+  100% {
+    transform: translate3d(0, 0, 0);
+  }
+  50% {
+    transform: translate3d(-60%, 0, 0);
+  }
+  90% {
+    transform: translate3d(5px, 0, 0);
+  }
+}
+
 // enable icon hover animation and 
 // name visibility on desktops only
 
 @media (hover: hover) and (pointer: fine) {
   .dock-app-container {
     &:hover {
-      margin: 0px 15px;
-      .dock-app-name  {
+      .dock-app-icon-container {
+        .dock-app-notification {
+          transform: scale(1.3) translate(-1px, -17px);
```

Comment:
Shall we use `scale3d` and `translate3d` here for GPU acceleration?

Reply:
Yes. I have now updated all instances where I used `scale` and `transform` to `scale3d` and `translate3d`. I should have clued in to this from your last commit to `DockApp.vue`.